about.html

```diff
@@ -16,22 +16,55 @@
 
   <header>
     <navbar class="nav-dsk">
-      <a href="index.html"><img src="images/logo.png" alt="" class="logo"></a>
+      <a href="index.html"><img src="images/logo.png" alt="logo" class="logo"></a>
+      <div class="dark-nav">
+        <ul>
+          <li><a href="#"><i class="fa-brands fa-facebook-f"></i></a></li>
+          <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
       <ul>
-        <li><a href="index.html">Home</a></li>
         <li><a href="about.html">About</a></li>
+        <li><a href="about.html">Program</a></li>
+        <li><a href="about.html">Join</a></li>
+        <li><a href="about.html">Sponsors</a></li>
+        <li><a href="about.html">News</a></li>
         <li class="menu-highlight"><a href="#">Stoa Campaign</a></li>
       </ul>
     </navbar>
     <!-- Menu Mobile-->
-    <navbar class="nav"><img src="images/mobile_menu.png" alt="hamburger"></navbar>
+    <nav>
+      <a class="hamburger">
+        <img src="images/mobile_menu.png" alt="hamburger">
+      </a>
+    </nav>
+    <nav>
+
+      <ul class="mobile-menu">
+        <a class="close"><img src="images/cancel.png" alt=""></a>
+        <li class="menu-link"><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="about.html">Program</a></li>
+        <li><a href="about.html">Join</a></li>
```

Comment:
- [ ] please, like the previous reviewer noted, kindly add dividers to differentiate the 3 sections of the about page. 
 
yours
![Screenshot (887)_LI](https://user-images.githubusercontent.com/65298048/226122948-3afb3290-c25f-4897-9585-e6380e239157.jpg)
cc design
![Screenshot (888)_LI](https://user-images.githubusercontent.com/65298048/226123012-e73c556c-8135-4044-9fd0-7c2ea8e3c7ad.jpg)

Reply:
Ok, it seems like for some reason, my desktop screen doesn't allow me to see that for the brightness, I checked this review with my mobile and now Im able to see the separation, I will try to fix it now. Thank you.